tests/features/bootstrap/FeatureContext.php

```diff
@@ -162,6 +162,9 @@ public function iEnterTheOtp(): void
     public function iEnterTheSmsVerificationCode(): void
     {
         $cookieValue = $this->minkContext->getSession()->getDriver()->getCookie('smoketest-sms-service');
+        if ($cookieValue === null) {
+            throw new RuntimeException('Unable to load the smoketest-sms-service cookie');
+        }
```

Comment:
Todo, this case is now occurring as the current Behat setup does not use the SMS service test double. Which would set the cookie in the first place.

I created a chore to fix this another time.

https://www.pivotaltracker.com/story/show/187940465